public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
We need to change this as well right?

Reply:
Fixed by a66cbb117ca5bd1bc65d7fff6d176b7a71b932b9.